_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -49,81 +50,65 @@ _There are no major accessibility support issues known for this rule._
 
 #### Passed Example 1
 
-`aria-pressed` state is supported for role `button`.
+The `aria-pressed` [state][] is [supported][] for the [semantic role][] `button`, which is the [explicit role][] of this `div` element.
 
 ```html
 <div role="button" aria-pressed="false">My button</div>
 ```
 
 #### Passed Example 2
 
-`aria-pressed` state is supported for `role=button` that is the [implicit role](#implicit-role) for `button` element.
+The `aria-pressed` [state][] is [supported][] for the [semantic role][] `button`, which is the [implicit role][] for `button` elements.
 
 ```html
 <button aria-pressed="false">My button</button>
 ```
 
 #### Passed Example 3
 
-Global state that is supported by all base markup elements.
+The `aria-busy` [state][] is a [global][] [state][] that is [supported][] by all elements, even without any [semantic role][].
 
 ```html
 <div aria-busy="true">My busy button</div>
 ```
 
 #### Passed Example 4
 
-`aria-label` state is inherited for role `button`
+The `aria-label` [state][] is [inherited][] for the [semantic role][] `button`.
```

Comment:
This is kind of odd. aria-label is global, should we say that?

Reply:
It is global, which as far as role definition goes (https://www.w3.org/TR/wai-aria/#button) means that is is inherited or all roles (from the abstract top-level `roletype`).

The thing is if you look at the description of the `button` role (or any other), then the global states and properties are listed in the "inherited" box (which is true, global are supported by `roletype` and inehirted by the rest). So just saying it's global might be a bit confusing when looking at the specs and  seeing it As "inherited".

I tried to improve the description to explain that. Feel free to suggest something better.